src/main/modules/nunjucks/index.ts

```diff
@@ -101,6 +103,11 @@ export class Nunjucks {
     nunjucksEnv.addGlobal('UnemploymentType', UnemploymentType)
     nunjucksEnv.addGlobal('BankAccountType', BankAccountType)
     nunjucksEnv.addGlobal('ClaimStatus', ClaimStatus)
+    nunjucksEnv.addGlobal('AppPaths', AppPaths)
+    nunjucksEnv.addGlobal('AppPaths', AppPaths)
+    if (FeatureToggles.isEnabled('finePrint')) {
+      nunjucksEnv.addGlobal('cookieText', `GOV.UK uses cookies make the site simpler.<a href="${AppPaths.cookiesPage.uri}"> Find out more about cookies</a>`)
```

Comment:
Really small thing but it would more readable if we had `simpler. <a href="${AppPaths.cookiesPage.uri}">Find out more about cookies</a>`. The only change is having whitespace after dot instead of heaving it within <a> tag.

I do not mandate that change, will leave it to you.